packages/replay-next/components/sources/log-point-panel/Capsule.tsx

```diff
@@ -60,13 +60,8 @@ export default function Capsule({
   //
   // To prevent re-layout when stepping between the latter two states,
   // pre-calculate the max width of the capsule (e.g. "10/10").
-  const minLabelWidthCh = useMemo(() => {
-    if (tooManyPointsToFind) {
-      return "10k+".length;
-    } else {
-      return `${hitPoints.length}/${hitPoints.length}`.length;
-    }
-  }, [hitPoints.length, tooManyPointsToFind]);
+  const numeratorMinWidth = `${hitPoints.length}`.length;
+  console.log("numeratorMinWidth:", numeratorMinWidth);
```

Comment:
:hand: Kill the log :)

Reply:
Hehe, oops.

Looks like the screenshots caught a regression too: 
https://delta.replay.io/project/dcb5df26?action=a0f1868b-f9cc-4000-a79b-90d7ebdba8e8&branch=FE-1403&fileName=log-point-message-too-many-points-to-run-analysis.png&

I'll pick it up in the am.